src/plugin.js

```diff
@@ -9,17 +9,19 @@ export function install(Vue, options = {}) {
 }
 
 function registerComponents(Vue, components = {}, defaultConfigurations = {}) {
-  if (!components) { throw new Error("No component found.") }
+  if (!defaultConfigurations) { console.warn('🛑 There is no default configuration for Cloudinary found!') }
```

Comment:
Don't we need the no component error?

Reply:
No, it's redundant because if there is no components passed, it takes `{}` as default fallback. And if you look at the logic where `registerComponents` is used, there is no way a `components` with `null` or `undefined` or anything not working will be passed.

Reply:
awesome.